packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -221,6 +222,26 @@ const getAppealType = (appealCaseData) => {
 	return `${appealCaseData.appealTypeName} appeal`;
 };
 
+/**
+ * @param {object} appealCaseData return object from database call
+ * @returns {string | null }
+ */
+const getDecisionOutcome = (appealCaseData) => {
+	if (!appealCaseData.outcome) return null;
+	switch (appealCaseData.outcome) {
+		case DECISION_OUTCOME.ALLOWED:
+			return 'allowed';
+		case DECISION_OUTCOME.DISMISSED:
+			return 'dismissed';
+		case DECISION_OUTCOME.INVALID:
+			return 'notice varied and upheld';
```

Comment:
Based on https://pins-ds.atlassian.net/browse/AAPD-995 I think we drop this case